browse/static/js/catalyzex.js

```diff
@@ -57,6 +57,11 @@
       .append(codeLink)
   } else {
     $output.append(`<p>No code found for this paper just yet.</p>`)
+
+    const relatedCodeURL = new URL(`https://www.catalyzex.com/paper/arxiv:${arxivId}/code/related`);
+    if(paperTitle) relatedCodeURL.searchParams.set('paperTitle', paperTitle);
```

Comment:
small q, but given that paperTitle is set this way `const paperTitle = document.querySelector("h1.title")?.innerText;`, do we need the conditional check here?

Reply:
yes! otherwise, we could set the search param as `?paperTitle=undefined` or `?paperTitle=` in case we retrieve a falsy paper title. 

the first validation you mentioned is just to make sure I don't break the script in case the element `h1.title` is somehow unavailable and I try to access an invalid property (i.e. `innerText`).